### PR TITLE
Move SEAL to CMS-IG namespace (BC Break)

### LINKED ISSUE #428: Moving to a new namespace and package name
The project did begin as a research project and so I quickly created a `schranz-search` and used the `Schranz\Search\SEAL\` namespace. This will be changed for 1.0 release. Currently different namespaces are in discussions. An upgrade via `RenameNamespaceRector` or via Search and Replace in your codebase should be easy.

What these also means is that we will may also move the name of the `composer` package. This sadly will loose our packagist statistic, but better early as later.

FYI: @ker0x, @toflar

CURRENT UPGRADE INSTRUCTIONS can be found in the old organisation: https://github.com/schranz-search/

### PR DESCRIPTION
**Pull request by alexander-schranz**
fix #428

# BC Breaks

To upgrade use search and replace also have a look at the .examples changes to better understand the upgrade.

 - Package names:
    - `schranz-search/seal` -> `cmsig/seal`
    - `schranz-search/seal-...-adapter` -> `cmsig/seal-...-adapter`
    - `schranz-search/laravel-package` -> `cmsig/seal-laravel-package`
    - `schranz-search/mezzio-module` -> `cmsig/seal-mezzio-module`
    - `schranz-search/spiral-bridge` -> `cmsig/seal-spiral-bridge`
    - `schranz-search/yii-module` -> `cmsig/seal-yii-module`
    - `schranz-search/symfony-bundle` -> `cmsig/seal-symfony-bundle`

 - Namespaces:
    - `Schranz\Search\SEAL` -> `CmsIg\Seal\`
    - `Schranz\Search\Integration` -> `CmsIg\Seal\Integration`

 - Config Files:
    - `schranz_search.*` -> `cmsig_seal.*`

 - Config, Service names, Tagged Services:
    - `schranz_search.*` -> `cmsig_seal.*`

 - Commands:
    - `schranz:search:*` -> `cmsig:seal:*`

 - Spiral Integration Changes:
   - `SearchBootloader` -> `SealBootloader`
   - `SearchConfig` -> `SealConfig`
   - `seal.php` -> `cmsig_seal.php`

 - Symfony Integration Changes:
   - `SearchBundle` -> `SealBundle`
   - `seal.yaml` -> `cmsig_seal.yaml`

 - Laravel Integration Changes:
   - `SearchProvider` -> `SealProvider`
   - `seal.php` -> `cmsig_seal.php`

### CODE REVIEW

File: integrations/symfony/src/Command/IndexCreateCommand.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 /**
  * @experimental
  */
-#[AsCommand(name: 'schranz:search:index-create', description: 'Create configured search indexes.')]
+#[AsCommand(name: 'seal:index-create', description: 'Create configured search indexes.')]
```
Comment (alexander-schranz) [inline integrations/symfony/src/Command/IndexCreateCommand.php]: Unsure about this one `seal:index-create`, `cmsig-seal:index-create` vs `cmsig:seal:index-create`

File: integrations/symfony/src/SearchBundle.php [hunk 2]
```diff
@@ -32,7 +32,7 @@
  */
 final class SearchBundle extends AbstractBundle
 {
-    protected string $extensionAlias = 'schranz_search';
+    protected string $extensionAlias = 'seal';
```
Comment (alexander-schranz) [inline integrations/symfony/src/SearchBundle.php]: unsure about this based on `flysystem` bundle I would go with just `seal`. alternative would be `cmsig_seal` like it is for `fos_rest`

**Review by alexander-schranz — COMMENTED** [review]
What you think here @toflar @bmack

File: integrations/symfony/src/Command/IndexCreateCommand.php (hunk 1 above)
Comment (Toflar) [reply integrations/symfony/src/Command/IndexCreateCommand.php]: I like `cmsig:seal:*`. We also do this in Contao, e.g. `contao:backup:create` etc.

File: integrations/symfony/src/SearchBundle.php (hunk 2 above)
Comment (Toflar) [reply integrations/symfony/src/SearchBundle.php]: There's unlikely going to be another `seal` but `cmsig_seal` would be more namespace-likey.

Comment (bmack) [reply integrations/symfony/src/SearchBundle.php]: I think "seal" is good.

File: integrations/symfony/src/Command/IndexCreateCommand.php (hunk 1 above)
Comment (bmack) [reply integrations/symfony/src/Command/IndexCreateCommand.php]: + `cmsig:seal:`

Comment (alexander-schranz) [reply integrations/symfony/src/Command/IndexCreateCommand.php]: I go with `cmsig:seal:`

File: integrations/symfony/src/SearchBundle.php (hunk 2 above)
Comment (alexander-schranz) [reply integrations/symfony/src/SearchBundle.php]: to be consistent with command and namespace I also choiced here `cmsig_seal`

Comment (alexander-schranz) [reply integrations/symfony/src/SearchBundle.php]: personally think yii framework is doing the best, where module config name = composer.json name think more frameworks should adopt that.

**Pull request merged**